arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Ior.kt

```diff
@@ -35,10 +35,12 @@ public sealed class Ior<out A, out B> {
    *
    * Example:
    * ```
+ * <!--- KNIT example-ior-01.kt -->
```

Comment:
question: 
This is generating empty kotlin files without the example
Was this intended or rather something like?
```markdown

* ```kotlin
* Left("tulip").isRight           // Result: false
* Right("venus fly-trap").isRight // Result: true
* Both("venus", "fly-trap").isRight // Result: false
* ```
<!--- KNIT example-ior-01.kt -->
```

Reply:
Ye, that doesn't look right.

Reply:
Seems like Knit adds a bunch of empty file headers as this and we committed it since they compile. I'll do another pass today over empty files.

Reply:
Just pushed a bunch of knit fixes for the Ior file and others.